src/core/math/RbMathFunctions.cpp

```diff
@@ -716,12 +716,10 @@ double RbMath::incompleteGamma(double x, double alpha, bool regularized, bool lo
         gin *= factor / alpha;
     }
 
-    if ( lower = false )
-    {
-        gin = 1.0 - gin;
-    }
-
-    return regularized ? gin : exp(log(gin) + scale);
+    gin = lower ? gin : 1.0 - gin;
+    gin = regularized ? gin : exp(log(gin) + scale);
```

Comment:
Is there a reason here to do `exp(log(gin)+scale)` versus (say) `gin*exp(scale)`?

Reply:
It's just a one-liner log-sum-exp. It's to avoid under/overflow the event that gin is very small and/or scale is very large.

Reply:
I suppose I could add a catch for overflow.